sudoers.json

```diff
@@ -0,0 +1,158 @@
+            "Cmnd_Specs": [
+                {
+                    "runasusers": [
+                        { "username": "ALL" }
+                    ],
+                    "runasgroups": [
+                        { "usergroup": "ALL" }
+                    ],
+                    "Options": [
+                        { "setenv": true }
+                    ],
+                    "Commands": [
+                        { "command": "ALL" }
+                    ]
+                }
+            ]
+        },
+        {
+            "User_List": [
+                { "useralias": "SYSADMINS" }
+            ],
+            "Host_List": [
+                { "hostname": "ALL" }
+            ],
+            "Cmnd_Specs": [
+                {
+                    "runasusers": [
+                        { "username": "john" }
+                    ],
+                    "runasgroups": [
+                        { "usergroup": "sudo" }
+                    ],
+                    "Options": [
+                        { "authenticate": false },
+                        { "setenv": true }
+                    ],
+                    "Commands": [
+                        { "command": "/usr/bin/cat" }
+                    ]
+                },
+                {
+                    "runasusers": [
+                        { "username": "tom" }
+                    ],
+                    "Options": [
+                        { "authenticate": false },
+                        { "setenv": true }
+                    ],
+                    "Commands": [
+                        { "command": "/etc/shadow" }
+                    ]
+                }
+            ]
+        },
+        {
+            "User_List": [
+                { "usergroup": "admin" }
+            ],
+            "Host_List": [
+                { "hostname": "ALL" }
+            ],
+            "Cmnd_Specs": [
+                {
+                    "runasusers": [
+                        { "username": "ALL" }
+                    ],
+                    "Options": [
+                        { "setenv": true }
+                    ],
+                    "Commands": [
+                        { "command": "ALL" }
+                    ]
+                }
+            ]
+        },
+        {
+            "User_List": [
+                { "usergroup": "sudo" }
+            ],
+            "Host_List": [
+                { "hostname": "ALL" }
+            ],
+            "Cmnd_Specs": [
+                {
+                    "runasusers": [
+                        { "username": "ALL" }
+                    ],
+                    "runasgroups": [
+                        { "usergroup": "ALL" }
+                    ],
+                    "Options": [
+                        { "setenv": true }
+                    ],
+                    "Commands": [
+                        { "command": "ALL" }
+                    ]
+                }
+            ]
+        }
+    ]
+}
```

Comment:
Maybe move this file to the `config/` directory, or even maybe a subdirectory of that. Just to avoid files in the root directory of the project.